Legacy/util/src/main/java/org/bonej/geometry/Vectors.java

```diff
@@ -79,4 +81,30 @@ private Vectors() {}
 		}
 		return vectors;
 	}
+	
+	/**
+	 * Calculate the cross product of 3 Point3f's, which describe two vectors
+	 * joined at the tails. Can be used to find the plane / surface normal of a
+	 * triangle. Half of its magnitude is the area of the triangle.
+	 *
+	 * @param point0 both vectors' tails
+	 * @param point1 vector 1's head
+	 * @param point2 vector 2's head
+	 * @return cross product vector
+	 */
+	public static Point3f crossProduct(final Point3f point0,
+		final Point3f point1, final Point3f point2)
+	{
+		final double x1 = point1.x - point0.x;
+		final double y1 = point1.y - point0.y;
+		final double z1 = point1.z - point0.z;
+		final double x2 = point2.x - point0.x;
+		final double y2 = point2.y - point0.y;
+		final double z2 = point2.z - point0.z;
+		final Point3f crossVector = new Point3f();
+		crossVector.x = (float) (y1 * z2 - z1 * y2);
+		crossVector.y = (float) (z1 * x2 - x1 * z2);
+		crossVector.z = (float) (x1 * y2 - y1 * x2);
+		return crossVector;
+	}
```

Comment:
I know `org.scijava.vecmath` is still needed for 3DViewer and old marching cubes, but since ImageJ is not going to support it forever, and BoneJ has already started to use `org.joml.joml` for linear algebra, we should try to avoid adding code that uses the artefact. 

In some ways it would be better to convert the `Point3f` to `org.joml.Vector3f` and call the perfectly good `crossProduct()` there, but I realise that'd be a bit inefficient. 

Since nothing else than `ParticleAnalysis` needs the method, lets make it a `private static` member of that class, so that we can do away with the `Vectors` utility class asap.

Reply:
OK, good strategy, will do. Can convert the methods as ImageJ moves to SciView and joml. In the meantime as long as the legacy 3DViewer is there and works well and is convenient for users, it's a stable solution.